### ISSUE TITLE
Need update of AudioKit

### ISSUE THREAD

**Issue by kingnotwo** [Audiograins/IASIAR#1]
Would you please update code with AudioKit v4.3, the latest version of AudioKit. 

Thanks!

**Comment by Audiograins** [comment]
Yes, will be planning to update to latest versions when I resume working on this project. Thanks
